hack/tools.go

```diff
@@ -1,20 +1,18 @@
 //go:build tools
 
-/*
-Copyright 2020 The Kubermatic Kubernetes Platform contributors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
```

Comment:
I had to change this because we don't use block comments in default license template so it wasn't matching license in this file and hence added duplicate license everytime we ran `npm fix`.